Infrastructure/GhostNetwork.Content.MongoDb/MongoPublicationStorage.cs

```diff
@@ -153,7 +159,15 @@ private static Publication ToDomain(PublicationEntity entity)
                 entity.Tags,
                 (UserInfo)entity.Author,
                 DateTimeOffset.FromUnixTimeMilliseconds(entity.CreateOn),
-                DateTimeOffset.FromUnixTimeMilliseconds(entity.UpdateOn));
+                DateTimeOffset.FromUnixTimeMilliseconds(entity.UpdateOn),
+                entity.Media.Select(ToDomainMedia));
```

Comment:
The same as for create, this line throws exception as old data doesn't have media field in db